pyconbalkan/timetable/models.py

```diff
@@ -0,0 +1,49 @@
+from django.db import models
+from pyconbalkan.core.models import ActiveModel
+from pyconbalkan.speaker.models import Speaker
+from django.db.models import CASCADE
+
+
+class Timetable(ActiveModel):
+    title = models.CharField(null=True, blank=True, max_length=100)
+    date = models.DateField(null=True, blank=True)
+
+    def __str__(self):
+        return self.title
+
+
+class Presentation(ActiveModel):
+    TALK = 0
+    WORKSHOP = 1
+    PRESENTATION_TYPE = (
+        (TALK, 'Talk'),
+        (WORKSHOP, 'Workshop'),
+    )
+
+    title = models.CharField(null=True, blank=True, max_length=100)
+    description = models.TextField(null=True, blank=True)
+    type = models.IntegerField(choices=PRESENTATION_TYPE, default=TALK)
+
+    speaker = models.ForeignKey(Speaker, blank=True, null=True, related_name='presentation', on_delete=CASCADE)
+
+    def __str__(self):
+        return '[{}] {}'.format(self.get_type_display(), self.title)
```

Comment:
@nadaj for choices field you can use `get_{field_name}_display` to display the field :) 🔝